src/app/component/ubs/components/ubs-submit-order/ubs-submit-order.component.scss

```diff
@@ -1,144 +1,55 @@
 .container {
   color: var(--ubs-primary-grey);
-  box-shadow: 1px 4px 8px rgb(100 114 125 / 18%);
   padding: 10px 25px 40px;
 }
 
-.header,
-.main {
-  width: 100%;
-  margin-top: 2rem;
-  margin-bottom: 3rem;
-}
-
-h3 {
-  font-weight: bold;
-}
-
-table {
-  width: 100%;
-  margin-top: 1rem;
-}
+.your-order {
+  background-color: #f8f9fb;
```

Comment:
If we don't have this color in styles.scss, make a variable on the top of this file.
Example: $yourOrderBackground: #f8f9fb;